pydrad/configure/configure.py

```diff
@@ -6,6 +6,7 @@
 import datetime
 import tempfile
 import shutil
+import platform
```

Comment:
Don't need this import anymore
```suggestion
```

Reply:
It's used in the `on_windows` function, unless you think that could be moved to util.py?

Reply:
That might actually be cleaner since it is used again in util.py...

Reply:
You only need the import in the file it's explicitly used, in this case `util.py`

Reply:
The `on_windows` function is in configure.py currently, so the import is needed.  What I meant is that perhaps `on_windows` should be moved to util.py.

Reply:
Sorry, I don't know what I'm thinking. 🤦‍♂️ You're right. 

Actually though, I think it would make more sense to move that function to `util.py` since it gets used in `run_shell_command`